src/Apps/Artist/Components/ArtistInsights/ArtistInsightPills.tsx

```diff
@@ -1,42 +1,40 @@
-import { Flex, Pill, Spacer } from "@artsy/palette"
+import { Flex, Pill } from "@artsy/palette"
 import { FC } from "react"
 import { createFragmentContainer, graphql } from "react-relay"
 import { ArtistInsightPills_artist$data } from "__generated__/ArtistInsightPills_artist.graphql"
-import { useScrollTo } from "Utils/Hooks/useScrollTo"
```

Comment:
should we use an eslint import restriction?

Reply:
I deleted the entire file.